seed.mts

```diff
@@ -232,6 +232,28 @@ async function generateSeed(): Promise<void> {
             order: 4,
         },
     ]);
+
+    await seed.statusOrders(
+        [
+            "No Status",
+            "Request Denied",
+            "Pending More Info",
+            "Called and Confirmed",
+            "Called and No Response",
+            "Shopping List Downloaded",
+            "Ready to Dispatch",
+            "Received by Centre",
+            "Collection Failed",
+            "Parcel Collected",
+            "Shipping Labels Downloaded",
+            "Out for Delivery",
+            "Delivered",
+            "Delivery Failed",
+            "Delivery Cancelled",
+            "Fulfilled with Trussell Trust",
+            "Request Deleted",
+        ].map((event_name, index) => ({ eventName: event_name, workflowOrder: index }))
```

Comment:
We could, but I think it would make more sense to keep the list here and get the statuses from the backend in Statuses.tsx
Could change it for now though if you want?

Reply:
yeah sounds good to me - no actions needed in this file then :)